server/routes/api/place.py

```diff
@@ -16,13 +16,21 @@
 import json
 import services.datacommons as dc
 
-from flask import Blueprint, request, jsonify
+from flask import Blueprint, jsonify, request, Response
 
 from cache import cache
 from services.datacommons import fetch_data
-from routes.api.stats import get_stats_wrapper
+import routes.api.stats as stats_api
 
-WANTED_PLACE_TYPES = [
+# Place types to keep for list of child places, keyed by parent place type.
+WANTED_PLACE_TYPES = {}
+WANTED_PLACE_TYPES['Country'] = [
+    "State", "Province", "County", "EurostatNUTS1", "EurostatNUTS2",
+    "AdministrativeArea1"
+]
+WANTED_PLACE_TYPES['State'] = ["Province", "County"]
```

Comment:
I want to spend a few minutes to lay out the choices here.

1. A place may have equivalent place types, which one should we pick from among them?
      - AA1/State/Province: pick AA1?   (this makes it general across countries)
      - County/AA2: pick AA2?
      - City/Village/Borough/Town: pick City?
      - Neighborhood
      - Zip Codes
      Note1: EurostatNUTS classification is separate from above, but may overlap (e.g. NUTS2 might AA1, NUTS3 might be AA2 for some geos)
      Note2: There are two general types "AdministrativeArea" and "Place" (this is often a City equivalent)

2. A place may have multiple types, some equivalent and some not. Do we pick one type for a place, or multiple (one for each equivalent set)?
      - if we pick multiple then the same node may occur twice in the children listing, which is odd.
      - were we to pick one place, the preference list should be in kPlaceTypesWithPopObs.

3. For a given parent type, which child types to pick/drop?
      - Looks like this is what WANTED_PLACE_TYPES does. But I suspect the key should consider all State-equivalent types, County-equivalent types?

4. For a parent node, do we group geos by "equivalent" types?
      - If one place had a type AA2 and another had County we should probably group them.

Reply:
Chatted offline, what we'll do is:

1. ignore AAx for non US, State/County/City for US. Eurostat* will stay separate

2. we'll show the place multiple times for non-equivalent types

3. pick a preferred type from an equivalent set, before key'ing off the list. so the list should have all equivalent place types
Equivalent types for US (geoId/... or zip/...)
State == AA1
County == AA2
City == Town / Village / Burrough
Everywhere else, pick AAx

4. For non-US geos, ignore Place and AA if parent AA2 onwards - group under Place

Reply:
Thanks for the summary.

4. To confirm, we want to ignore child AA + Place for parent AA1 and Country, but keep for parent AA2, AA3, ...?

Reply:
will do this in a follow up!